tests/unit/field/test_field_factory.cxx

```diff
@@ -24,12 +24,28 @@ template <typename T>
 class FieldFactoryCreationTest : public FakeMeshFixture {
 public:
   FieldFactoryCreationTest() : FakeMeshFixture{}, factory{mesh} {
-    // We need a parallel transform as FieldFactory::create3D wants to
-    // un-field-align the result
-    mesh->setParallelTransform(
+    // We need Coordinates so a parallel transform is available as
+    // FieldFactory::create3D wants to un-field-align the result
+    static_cast<FakeMesh*>(mesh)->setCoordinates(std::make_shared<Coordinates>(
```

Comment:
It looks like we're remaking the exact same `shared_ptr<Coordinates>` object several times here. I'd probably just make it once and save it as a member of the test fixture, and then pass that to `setCoordinates`